Frontend/ui-common/src/Engineering/Helpers/Utils.js

```diff
@@ -57,7 +60,7 @@ export const incrementMonth = function (month) {
   return function (year){
     try{
       const date = new Date(year, month+1, 1);
-      let d= { utcDate: date.toISOString(), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: date.getFullYear(), intMonth : date.getMonth(),
+      let d= { day: getISODate(date), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: date.getFullYear(), intMonth : date.getMonth(),
```

Comment:
Update the name for `let d` here to something understandable

Reply:
done